2022/day01/luigibarbato/parts_test.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"testing"
+)
+
+var sample string = ``
+
+func TestPartOne(t *testing.T) {
+	tt := []struct {
+		name     string
+		input    []string
+		expected int
+	}{
+		{
+			name: "Successfull test",
+			input: []string{
+				"1000",
+				"2000",
+				"3000",
+				"",
+				"4000",
+				"",
+				"5000",
+				"6000",
+				"",
+				"7000",
+				"8000",
+				"9000",
+				"",
+				"10000",
+				"",
+			},
```

Comment:
Not sure why you dropped the `utils.SplitInput(tc.input)`. It's handy I think. You can just copy-paste the sample input: https://www.youtube.com/watch?v=sxkQToahs4M&t=688s

https://github.com/golangroma/aoc/blob/main/2022/day01/enrichman/parts_test.go